Tests/Test_Renode_CANBPSFault.c

```diff
@@ -28,7 +28,7 @@ void Task1(void *p_arg){
     // Enable contactors for ReadCarCAN to flip them
     Contactors_Init();
     Contactors_Enable(ARRAY_CONTACTOR);
```

Comment:
Shouldn't this be the motor precharge bypass contactor?

Reply:
I think this test can be deleted because it references fault state.